src/main/resources/marc4bibframe.yml

```diff
@@ -1644,17 +1644,23 @@ bibFieldRules:
           multiply: true
           subfields:
             a: NAME
+          ld2marcCondition:
```

Comment:
Instead of specifying ld2marcCondition -> skip: true for each edge, can we define it once for the 776 field? ie, at line 1639. Will that work?

Reply:
No PK, it will not work with current implementation, tried to do this